tests/queueing_honey_badger.rs

```diff
@@ -93,9 +93,10 @@ fn new_queueing_hb(
     let mut rng = rand::thread_rng().gen::<Isaac64Rng>();
     let (qhb, qhb_step) = QueueingHoneyBadger::builder(dhb)
         .batch_size(3)
-        .build(&mut rng);
+        .build(&mut rng)
+        .expect("failed to build QueueingHoneyBadger");
```

Comment:
Technically, we could use a result here, but it's a legacy test, so... =)

Reply:
Yeah, as if the test was a (traditional) main function.

Reply:
Right; I don't think it's worth updating all the legacy tests for now?

Reply:
Nope =).